setup-lesson-deps/action.yaml

```
@@ -31,15 +31,15 @@ runs:
           if (on_linux) {
             if (Sys.getenv("RSPM") == "") {
               release <- system("lsb_release -c | awk '{print $2}'", intern = TRUE)
-              Sys.setenv("RSPM" = 
-                paste0("https://packagemanager.rstudio.com/all/__linux__/", release, "/latest"))
+              Sys.setenv("RSPM" =
+                paste0("https://packagemanager.posit.co/all/__linux__/", release, "/latest"))
```

```suggestion
                paste0("https://packagemanager.posit.co/all/__linux__/", release, "/latest")) 
```

suggesting superfluous whitespace to get add a verified commit